Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Generator.cs

```diff
@@ -134,14 +134,26 @@ public void Execute(GeneratorExecutionContext context)
                     }
                 }
 
-                var configureMethodSymbol = semanticModelProvider.GetConfigureMethodModel(receiver.StartupClasses.FirstOrDefault());
+                var configureHostBuilderMethodSymbol = semanticModelProvider.GetConfigureHostBuilderMethodModel(receiver.StartupClasses.FirstOrDefault());
+                var configureServicesMethodSymbol = semanticModelProvider.GetConfigureServicesMethodModel(receiver.StartupClasses.FirstOrDefault());
+                var configureMethodSymbol = configureServicesMethodSymbol;
+
+                if (configureServicesMethodSymbol != null && configureHostBuilderMethodSymbol != null)
+                {
+                    diagnosticReporter.Report(Diagnostic.Create(DiagnosticDescriptors.MultipleConfigureMethodsNotAllowed, Location.None, configureServicesMethodSymbol.Name, configureHostBuilderMethodSymbol.Name));
```

Comment:
Instead of `Location.None` what do you think about `configureServicesMethodSymbol.Locations.FirstOrDefault()` so we can possibly give users a place to click on the error message and go to. I guessing `configureServicesMethodSymbol` since that is the older one and we probably want to steer people towards the newer method.

Reply:
yea i like that idea

Reply:
updated in latest commit and i retested and clicking take me to ConfigureServices as expected